pytest.ini

```diff
@@ -0,0 +1,9 @@
+[pytest]
+python_files = *.py
+norecursedirs = .* _* node node_modules coverage venv
+addopts =
+    --tb short
+    --cov=metashare
+    --cov-report html
+    --cov-report term
```

Comment:
Half the time I have to remove this line while working on a subset of tests or a known-failing test, as it causes the coverage report to be so long that it pushes the failure into the scrollback. It's useful when I expect the tests to all pass and want to see that 100%, but I might remove it by default here.